
class Sorry(Exception):
    pass

class Failure(Exception):
    pass
